### PR TITLE
prepare for publishing on crates.io

### PR DESCRIPTION
**Pull request by pmnoxx**
Some changes are required to publish on crates.io

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2017 Actix Team
```
Comment (bowenwang1996) [inline LICENSE-MIT:1]: this part should be changed

File: near-rust-allocator-proxy/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,14 @@
 [package]
 name = "near-rust-allocator-proxy"
-version = "0.2.6"
+version = "0.2.7"
 authors = ["Piotr Mikulski <piotr@near.org>"]
```
Comment (bowenwang1996) [inline near-rust-allocator-proxy/Cargo.toml:4]: @nearmax what is our policy regarding authors?

**Review by bowenwang1996 — COMMENTED** [review]
@nearmax does this license look good to you

File: LICENSE-MIT (hunk 1 above)
Comment (pmnoxx) [reply LICENSE-MIT]: I removed that file

File: near-rust-allocator-proxy/Cargo.toml (hunk 2 above)
Comment (pmnoxx) [reply near-rust-allocator-proxy/Cargo.toml]: I changed authors to be
```
authors = ["Near Inc <hello@nearprotocol.com>", "Piotr Mikulski <piotr@near.org>"]
```

Comment (pmnoxx) [reply near-rust-allocator-proxy/Cargo.toml]: @nearmax @bowenwang1996  Is there anything else stopping us from publishing this crate?

Comment (bowenwang1996) [reply near-rust-allocator-proxy/Cargo.toml]: I don't think so. Please go ahead and publish the crate

**Review by bowenwang1996 — APPROVED** [review]

File: near-rust-allocator-proxy/Cargo.toml (hunk 2 above)
Comment (nearmax) [reply near-rust-allocator-proxy/Cargo.toml]: This should be fine.

**Pull request merged**